pep-0683.rst

```diff
@@ -724,7 +724,8 @@ Other (less practical) solutions:
 * only do it if a stable-ABI extension has been imported
 * provide a runtime flag for disabling immortality
 
-Also see: https://mail.python.org/archives/list/python-dev@python.org/message/OXAYWH47ZGLOWXTNKCIW4YE5PXGHNT4Y/
+(`The discussion thread <https://mail.python.org/archives/list/python-dev@python.org/message/OXAYWH47ZGLOWXTNKCIW4YE5PXGHNT4Y/>`
```

Comment:
This hyperlink reference is missing its trailing underscore(s), but strangely still renders correctly without errors or warnings in Sphinx despite the [official Docutils reST spec](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#hyperlink-references) stating that only "standalone links" (i.e. raw/literal URIs) omit it.

I'm baffled and can't find any authoritative information on this; perhaps this is something that Sphinx does differently than docutils that isn't documented anywhere I could find—@AA-Turner any insight here? Is it treated as an anonymous reference, named reference or something else?